docs/components/segmented-control.md

```diff
@@ -118,3 +180,58 @@ export default defineComponent({
 })
 </script>
 ```
+
+## Theming
+
+| Variable | Purpose
+|:-------- |:-------
+| `--KSegmentTextColor` | Option text color
```

Comment:
question: since we don't use the `Color` suffix in any of the other variables, should this just be `--KSegmentText` instead?

If not, then as the inverse, I'd recommend adding a suffix of `*Color` to all the variables below.